README.md

```diff
@@ -64,6 +64,16 @@ copy(
 
 Update the index page & copyright page URLs.
 
+## Create the SQLite Index
+
+Create a SQLite database in the file `PeopleTools.docset/Contents/Resources/docSet.dsidx`:
+
+`sqlite3 PeopleTools.docset/Contents/Resources/docSet.dsidx`
```

Comment:
Instead of making them three lines of code (which display inline, one after the other), wrap all three in a code block (three backticks at the top and bottom).

e.g.
````
```
sqlite3 PeopleTools.docset/Contents/Resources/docSet.dsidx
CREATE TABLE SearchIndex(id INTEGER PRIMARY KEY, name TEXT, type TEXT, path TEXT);
CREATE UNIQUE INDEX anchor ON SearchIndex (name, type, path);
```
````

Reply:
Sounds good, just pushed a commit with the suggestion